web/controllers/api/owner_controller.ex

```diff
@@ -27,9 +27,18 @@ defmodule HexWeb.API.OwnerController do
   def create(conn, %{"email" => email}) do
     email = URI.decode_www_form(email)
     user = HexWeb.Repo.get_by!(User, email: email)
+    package = conn.assigns.package
+    owners = assoc(package, :owners) |> HexWeb.Repo.all
```

Comment:
it was already sending emails to new owner, but yeah, doing the query after creating would be a little bit cleaner